lib/ILinkWeightCalculator.php

```diff
@@ -33,8 +33,8 @@
 
 interface ILinkWeightCalculator {
 	/**
-	 * @param IRelatedResource[] $paths
+	 * @param IRelatedResource $current
```

Comment:
```suggestion
```
No need to duplicate this when it's already in the function declaration

Reply:
we already had a discussion about this, we keep phpdoc